templates/SilverStripe/Admin/Includes/BrowserWarning.ss

```diff
@@ -0,0 +1,31 @@
+<!--[if lte IE 9]>
+    <link rel="stylesheet" href="{$BaseURL}silverstripe-admin/client/dist/styles/browser-warning.css">
```

Comment:
I remember there was a `<% require css("cms/css/TreeSelector.css") %>` syntax you could use, similar to the javascript one below

Reply:
I think that's fine for this css file, it goes hand-in-hand with the javascript file